lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -447,6 +417,55 @@ void TypeLoweringVisitor::lowerBlock(Block *block) {
   }
 }
 
+ArrayAttr TypeLoweringVisitor::filterAnnotations(
+    MLIRContext *ctxt, ArrayAttr annotations, FIRRTLType srcType,
+    FlatBundleFieldEntry field, bool &hasDontTouch, StringAttr sym) {
+  hasDontTouch = false;
```

Comment:
filtering seems a generic operation.  Filtering and testing for and removing another well known name seems a weird structure.

Reply:
Fixed it.